students/synowiec_krzysztof/lesson_05_lists/swap_min_and_max.py

```diff
@@ -0,0 +1,20 @@
+def main():
+    list = [int(x) for x in input().split()]
+    minIndex = 0
+    maxIndex = 0
+    for x in range(1, len(list)):
+        if list[x] > list[maxIndex]:
+            maxIndex = x
+            continue
```

Comment:
does `continue` do anything here? There is no more code in `for` block after `if` block. So the flow will be exactly the same without it. Then this just add complexity where it is not necessary.
Bad move :)